plugins/org.obeonetwork.m2doc.genconf.edit/src-gen/org/obeonetwork/m2doc/genconf/provider/GenerationItemProvider.java

```diff
@@ -287,6 +310,7 @@ public void notifyChanged(Notification notification) {
             case GenconfPackage.GENERATION__TEMPLATE_FILE_NAME:
             case GenconfPackage.GENERATION__RESULT_FILE_NAME:
             case GenconfPackage.GENERATION__TIME_STAMPED:
+            case GenconfPackage.GENERATION__REFRESH_DIAGRAM:
```

Comment:
This and all the related stuff should be 'REFRESH_REPRESENTATION' because 'diagram' here is incorrect.